microsoftgraph/microsoft-graph-devx-api
# Task/tour content

## Overview
Adds an endpoint for Graph Explorer product tour content (/toursteps)

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes
The actual content will change as well as tests that verify the total number of steps. 

## Testing Instructions
Run the solution then type this url on your browser: https://localhost:44399/toursteps. This should fetch content from the blob storage
For github tests, type the url on your browser: https://localhost:44399/toursteps?org=microsoftgraph&branchName=dev
